src/user/client.ts

```diff
@@ -9,16 +11,19 @@ export interface User {
   // calendar: Calendar;
 }
 
-// TODO: Fill this in with methods
+function saveUserToDb(user: firebase.User) {
```

Comment:
I think it might be a good idea to create a separate `User` type instead of using the Firebase one — that way we can tailor the structure to our app's requirements. What do you think?

Reply:
yeah agreed.